src/Server/Connection.php

```diff
@@ -105,6 +105,11 @@ protected function processMessage(string $data)
         }
     }
 
+    public function disconnect()
```

Comment:
I'm in favor of adding a new listener on('end') in the method `initListeners` of this class. Because having a method `Connection::disconnect` that does not actually disconnect is not fluid for users.